src/modules/Task.js

```diff
@@ -0,0 +1,41 @@
+export default class Task {
+  constructor(description = '', index = 0) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+
+  set setDescription(newDescription = '') {
+    if (!newDescription) return;
+    this.description = newDescription;
+  }
+
+  set setCompleted(newCompleted) {
+    this.completed = newCompleted;
+  }
+
+  set setIndex(newIndex = 0) {
+    if (typeof newIndex !== 'number' || newIndex < 0) return;
+    this.index = newIndex;
+  }
+
```

Comment:
- [ ] Kindly make sure by default, new tasks have the property `index` set to the value of the new array length (i.e. if a 1st task is added to the list, the index of that task should equal 1; if a 2nd is added its index should be 2,...), right now your `index` property starts from **0, 1, 3,...**. Make sure to fix that.